valhalla/mjolnir/osmway.h

```diff
@@ -18,13 +18,17 @@ struct OSMWay {
    * Set way id.
    * @param   id  way id
    */
-  void set_way_id(const uint64_t id);
```

Comment:
i was really hoping the diff here was going to be to just remove all of these setters/getters to direct members because they are completely superfluous given that this is a simple struct. :frowning_face:

Reply:
(lots of changes may be needed so might not do it with this PR). I like any setters that do bounds checking so would want to keep those I think?